front/src/common/BootstrapSNCF/MultiSelectSNCF.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useEffect, useMemo, useState } from 'react';
+
+type MultiSelectSNCFProps = {
+  multiSelectTitle: string;
+  multiSelectSubTitle?: string;
+  selectOptions: { label: string }[];
+  onChange: React.Dispatch<React.SetStateAction<string[]>>;
+  selectedValues: string[];
+};
+
+const MultiSelectSNCF = ({
+  multiSelectTitle,
+  multiSelectSubTitle,
+  selectOptions,
+  onChange,
+  selectedValues,
+}: MultiSelectSNCFProps) => {
+  const [multiSelectToggle, setMultiselectToggle] = useState<boolean>(false);
+  const [selectAll, setSelectAll] = useState<boolean>(false);
+  const renderOptions = (options: { label: string }[]) =>
```

Comment:
use a callback or Memo function (I think callback function is better) to avoid regenerate the options at each render :)
```suggestion
  const renderOptions = (options: string[[]) =>
```

Reply:
Done